localstack/aws/protocol/serializer.py

```diff
@@ -1501,7 +1501,6 @@ def _serialize_response(
                 request_id,
             )
         self._serialize_content_type(response, shape, shape_members, mime_type)
-        self._prepare_additional_traits_in_response(response, operation_model, request_id)
```

Comment:
I removed this from the S3ResponseSerializer, because `_prepare_additional_traits_in_response` is already called once in `localstack.aws.protocol.serializer.ResponseSerializer.serialize_to_response`: it first calls `_serialize_response` then `_prepare_additional_traits_in_response`.